api/src/main/java/org/eclipse/microprofile/metrics/Metadata.java

```diff
@@ -218,8 +233,9 @@ public Metadata(String name, String displayName, String description, MetricType
      * <li>{@code type} - The type of the metric</li>
      * <li>{@code unit} - The units of the metric</li>
      * <li>{@code tags} - The tags of the metric  - cannot be null</li>
+     * <li>{@code reusable} - The metric name can be registered multiple times if the value is <tt>true</tt></li>
```

Comment:
This makes it sound like the registry would ultimately have multiple copies of metrics with the same name.  I'd suggest... "If true, this metric name is permitted to be used at multiple registration points.  If false, this metric name is only permitted to be used at one registration point per MetricRegistry."